### PR TITLE
ci: multinode scenario

### PR DESCRIPTION
**Pull request by leseb**
Signed-off-by: Sébastien Han <seb@redhat.com>

**Description of your changes:**

**Which issue is resolved by this Pull Request:**
Resolves #

**Checklist:**

- [ ] **Commit Message Formatting**: Commit titles and messages follow guidelines in the [developer guide](https://rook.io/docs/rook/master/development-flow.html#commit-structure).
- [ ] **Skip Tests for Docs**: Add the flag for skipping the build if this is only a documentation change. See [here](https://github.com/rook/rook/blob/master/INSTALL.md#skip-ci) for the flag.
- [ ] **Skip Unrelated Tests**: Add a flag to run tests for a specific storage provider. See [test options](https://github.com/rook/rook/blob/master/INSTALL.md#test-storage-provider).
- [ ] Reviewed the developer guide on [Submitting a Pull Request](https://rook.io/docs/rook/master/development-flow.html#submitting-a-pull-request)
- [ ] Documentation has been updated, if necessary.
- [ ] Unit tests have been added, if necessary.
- [ ] Integration tests have been added, if necessary.
- [ ] Pending release notes updated with breaking and/or notable changes, if necessary.
- [ ] Upgrade from previous release is tested and upgrade user guide is updated, if necessary.
- [ ] Code generation (`make codegen`) has been run to update object specifications, if necessary.

[skip ci]

### CODE REVIEW

File: cluster/examples/kubernetes/ceph/operator.yaml [hunk 1]
```diff
@@ -387,7 +387,7 @@ spec:
 
         # The logging level for the operator: INFO | DEBUG
         - name: ROOK_LOG_LEVEL
-          value: "INFO"
+          value: "DEBUG"
```
Comment (BlaineEXE) [inline cluster/examples/kubernetes/ceph/operator.yaml:390]: Intentional?

File: pkg/operator/ceph/cluster/osd/spec.go [hunk 2]
```diff
@@ -182,6 +182,9 @@ fi
 if [ -n "$VAULT_CLIENT_KEY" ]; then
 	ARGS+=(--key "${VAULT_CLIENT_KEY}")
 fi
+if [ -n "$VAULT_TLS_SERVER_NAME" ]; then
+	ARGS+=(--resolve "${VAULT_TLS_SERVER_NAME}")
+fi
```
Comment (BlaineEXE) [inline pkg/operator/ceph/cluster/osd/spec.go:187]: Is this related to the CI changes?

File: cluster/examples/kubernetes/ceph/operator.yaml (hunk 1 above)
Comment (BlaineEXE) [reply cluster/examples/kubernetes/ceph/operator.yaml:390]: FWIW, I just execute a command after deploying the operator in my dev environment:
`kubectl --namespace rook-ceph set env deployment/rook-ceph-operator ROOK_LOG_LEVEL=DEBUG`

File: pkg/operator/ceph/cluster/osd/spec.go (hunk 2 above)
Comment (leseb) [reply pkg/operator/ceph/cluster/osd/spec.go:187]: No, I don't remember how it got here

**Comment by leseb** [comment]
Investigate potential use of https://k3s.io/, https://github.com/k3s-io/k3s/blob/master/README.md, https://www.suse.com/c/introduction-k3d-run-k3s-docker-src/

**Comment by leseb** [comment]
W will re-open once the need increases but it does not seem feasible in our CI right now.

**Pull request closed without merging**